File: articles/algorithmic-harm-error-and-institutional-responsibility/cpp/harm_risk_score.cpp
#include <iostream>

int main() {
    double error_likelihood = 0.34;
    double severity = 0.92;
    double exposure = 0.78;
    double contestability = 0.42;
    double harm_risk = error_likelihood * severity * exposure * (1.0 - contestability);
    std::cout << "harm_risk_score=" << harm_risk << "\n";
    return 0;
}

File: articles/algorithmic-harm-error-and-institutional-responsibility/prolog/harm_risk_score.pl
harm_risk_score(ErrorLikelihood, Severity, Exposure, Contestability, HarmRisk) :-
    HarmRisk is ErrorLikelihood * Severity * Exposure * (1 - Contestability).

% Query example:
% ?- harm_risk_score(0.34, 0.92, 0.78, 0.42, HarmRisk).
